strapi/strapi
# [Blocks editor] change the label value for the ordered and unordered list

### What does it do?

- Change the label value for the ordered and unordered list to reflect the design (Numbered and Bulleted list)

### Why is it needed?

Because we want to respect the design provided.

### How to test it?

- Create a new blocks field inside a Collection.
- Hover on the ordered list and unordered list icons, you can see the labels "Bulleted list" (for the unordered list) and "Numbered list" (for the ordered list)
